npm/package.json

```diff
@@ -25,7 +25,8 @@
     "build": "npm run generate && npm run build:tsc",
     "generate": "node generate_katas_content.js && node generate_samples_content.js",
     "build:tsc": "node ../node_modules/typescript/bin/tsc -p ./src/tsconfig.json",
-    "tsc:watch": "node ../node_modules/typescript/bin/tsc -p ./src/tsconfig.json --watch"
+    "tsc:watch": "node ../node_modules/typescript/bin/tsc -p ./src/tsconfig.json --watch",
+    "test": "node --test"
```

Comment:
Is typing `npm run test` easier than `node --test`? :-)

Reply:
lol, it's actually `npm test` so I really did find it easier!! If there was a reason it was omitted, I can take it out, but I figured it was harmless.